Gemfile.lock

```diff
@@ -925,7 +925,7 @@ GEM
       addressable (>= 2.3.6)
       crack (>= 0.3.2)
       hashdiff
-    webpacker (3.5.5)
+    webpacker (3.6.0)
```

Comment:
In package.json webpacker is set to ^4.0.2, but maybe that's correct? Sorry for possible stupid comments, I'm not familiar with Ruby

Reply:
I totally made a mistake here. Nice catch!